cypress/integration/mocked/okta_change_password.2.cy.ts

```diff
@@ -133,7 +133,7 @@ describe('Change password in Okta', () => {
       mockPasswordResetInvalidStateTokenFailure();
       mockValidateRecoveryTokenFailure();
 
-      cy.visit(`/reset-password/token?useOkta=true`);
+      cy.visit('/reset-password/token?useIdapi=true');
```

Comment:
The tests in this file relate to okta, so they shouldn't have the `useIdapi=true` flag